From: Felipe Sateler <fsateler@debian.org>
Date: Fri, 25 Oct 2019 08:34:59 -0300
Subject: qpaeq: Drop unused imports

Neither sip nor PyQt5 are used. From PyQt5 we only use the named imports just below
Forwarded: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/merge_requests/197
---
 src/utils/qpaeq | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/utils/qpaeq b/src/utils/qpaeq
index ac0daec..914aedf 100755
--- a/src/utils/qpaeq
+++ b/src/utils/qpaeq
@@ -18,7 +18,6 @@
 
 import os,math,sys
 try:
-    import PyQt5,sip
     from PyQt5 import QtWidgets,QtCore
     import dbus.mainloop.pyqt5
     import dbus
